infra/machines/vcsa/create-esxi-vm.sh

```diff
@@ -29,8 +29,8 @@ EOF
 
 disk=48
 mem=16
-# 6.5.0a - http://pubs.vmware.com/Release_Notes/en/vsphere/65/vsphere-esxi-650a-release-notes.html
-iso=VMware-VMvisor-6.5.0-4887370.x86_64.iso
+# 6.5.0b - http://pubs.vmware.com/Release_Notes/en/vsphere/65/vsphere-vcenter-server-650b-release-notes.html
+iso=VMware-VMvisor-6.5.0-5146846.x86_64.iso
```

Comment:
Yes indeed.  updated the vcsa script, but looks like ESX is still the 6.5b build